README.md

```diff
@@ -89,6 +89,7 @@ Internals
 This library has no support for fragmented files or directories. File names
 are in 8.3 format: 8 chars followed by 3 for an extension. Filenames will be
 padded to 11 chars by spaces.
+You can change the filename size by modifying the fileName_length variable from 11 to the number of characters you need.
```

Comment:
As mentioned elsewhere, I'd rather not change the readme and keep this a purely internal change.